src/cf-forms/src/atoms/text-input.less

```diff
@@ -0,0 +1,56 @@
+.a-text-input {
+    .u-webfont-regular();
+
+    // Browser Reset
+    appearance: none;
+
+    display: inline-block;
+
+    padding: unit( 6px / @base-font-size-px, em );
+    border: 1px solid @input-border;
+
+    background: @input-bg;
+
+    &:focus,
+    &__focused {
+        border-color: @input-border__focused;
+        outline: 1px solid @input-border__focused;
+    }
+
+    &:disabled,
+    &__disabled {
+        background-color: @input-bg__disabled;
+        cursor: not-allowed;
+    }
+
+    &__error {
+        border-color: @input-border__error;
+        outline: 1px solid @input-border__error;
+    }
+
+    &__warning {
+        border-color: @input-border__warning;
+        outline: 1px solid @input-border__warning;
+    }
+
+    &__success {
+        border-color: @input-border__success;
+        outline: 1px solid @input-border__success;
+    }
+}
+
+// Overrides extra left padding.
```

Comment:
The following were already set up like this and I wasn't sure if they should be chained to text inputs specifically or left open like this. I'm open to suggestions.